bot/modules/speech.js

```diff
@@ -18,7 +18,12 @@ exports.speech = {
 	description: "gets top claim from spee.ch, coming soon post to spee.ch",
 	process: function(bot,msg,suffix){
 		
-		var ChannelID = "363085078403874823"
+		var ChannelID1 = "363085078403874823"
```

Comment:
These should be moved to an array inside of the config called speech with the channels as an array.